Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/Straight_skeleton_2.h

```diff
@@ -5,26 +5,182 @@ namespace CGAL {
 
 \cgalModels `StraightSkeleton_2`
 
-The class `Straight_skeleton_2` provides a model for the
-`StraightSkeleton_2` concept which is the class
-type used to represent a straight skeleton.
+The class `Straight_skeleton_2` provides a model for the `StraightSkeleton_2` concept
+which is the class type used to represent a straight skeleton.
 
-The only purpose of this class is to protect all the modifying
-operations in a `HalfedgeDS`. Normal users should not modify a
-straight skeleton. If an advanced user needs to get access to the
-modifying operations, it must call the required methods through the
-`Base` class.
+The only purpose of this class is to protect all the modifying operations in a `HalfedgeDS`.
+Normal users should not modify a straight skeleton. If an advanced user needs to get access to the
+modifying operations, it must call the required methods through the `Base` class.
 
-\tparam Traits_ must be a model of `Kernel`
+\tparam Traits must be a model of `Kernel`
+
+\cgalHeading{Angular Bisecting Lines and Offset Bisectors}
+
+Given two points and a line passing through them, the perpendicular line passing through the midpoint
+is the bisecting line (or bisector) of those points.
+
+Two non-parallel lines, intersecting at a point, are bisected by two other lines passing through
+that intersection point.
+
+Two parallel lines are bisected by another parallel line placed halfway in between.
+
+Given just one line, any perpendicular line can be considered the bisecting line (any bisector
+of any two points along the single line).
```

Comment:
it is a definition of the bisector of a line. It could also be that any line is perpendicular to the line splits the line equally so it is a bisector.